gradle.properties

```diff
@@ -1,5 +1,5 @@
 GROUP=com.dropbox.dropshots
-VERSION_NAME=0.3.1-SNAPSHOT
+VERSION_NAME=0.3.0-SNAPSHOT
```

Comment:
Did you mean to bump this back down to 0.3.0?

Reply:
Yep, the previous 0.3.0 release failed.